config.yaml

```diff
@@ -29,7 +29,7 @@
 
   # ------------------------------------------
   # Experiment specific info - REQUIRED
-  EXPERIMENT_SET : [experiment_name] # list of the experiments to be executed
+  EXPERIMENT_SET : ["PUM2_K562_ENCSR661ICQ_2"] # list of the experiments to be executed
```

Comment:
Is this the name of the directory? Should be more explicit about what "experiments to be executed" means.